src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -266,6 +266,9 @@ def _create_database_account(client,
     backup_policy = None
     if backup_policy_type is not None:
         if backup_policy_type.lower() == 'periodic':
+            if backup_interval is None and backup_retention is None:
+                raise CLIError(
+                    '--backup-interval and --backup-retention must be specified when Backup Policy Type is Periodic.')
```

Comment:
CC @amisi01 since I'm not familiar with service logic

Reply:
if backup_interval is None and backup_retention is None and backup_redundancy is None:
        raise CLIError(
                    '--backup-interval, --backup-retention or --backup-redundancy must be specified when Backup Policy Type is Periodic.')

Reply:
Made all the suggestions